test/test_fsspec.py

```diff
@@ -64,6 +64,15 @@ def test_fsspec_file_lister_iterdatapipe(self):
                 {fsspec.implementations.local.make_path_posix(file) for file in self.temp_sub_files},
             )
 
+        # checks for functional API
+        datapipe = IterableWrapper([self.temp_sub_dir.name])
+        listed = list(datapipe.list_file_by_fsspec())
+        for path in listed:
```

Comment:
Super nit:
```suggestion
        datapipe = datapipe.list_file_by_fsspec()
        for path in datapipe:
```

Reply:
https://github.com/pytorch/data/pull/463/commits/c51297a767b34e04edf25534f27ac3739ab15038

Reply:
Oh no, I mean you don't need to call `list` here. You can directly iterate over `datapipe`

Reply:
Ah, didn't notice. Will change it

Reply:
https://github.com/pytorch/data/pull/463/commits/a84ba1ea51a59634ffadba53e87634b3e0b4b24b